pkg/kv/kvserver/store.go

```diff
@@ -2193,7 +2193,6 @@ func (s *Store) GetConfReader(ctx context.Context) (spanconfig.StoreReader, erro
 
 	if s.cfg.SpanConfigsDisabled ||
```

Comment:
Should we get rid of all these knobs now that we're no longer gossipping the system config span: SpanConfigsDisabled, spanconfigstore.EnabledSetting, s.TestingKnobs().UseSystemConfigSpanForQueues.

Reply:
I swear I replied but the message disappeared. 

I tried removing it but some tests were failing and it wasn't obvious to me why. I added two more TODOs to remove `SpanConfigsDisabled` and `spanconfigstore.EnabledSetting` and you already had one for `s.TestingKnobs().UseSystemConfigSpanForQueues`